modules/ROOT/pages/manage-audit-logs.adoc

```diff
@@ -0,0 +1,292 @@
+
+|`"remote"`
+|Object
+|Contains key-value pairs for "ip" and outgoing "port", for the node on which the event-request was dispatched.
+
+|`"timestamp"`
+|ISO-8601 timestamp
+|UTC timestamp for when the event was generated.
+
+|`"optional fields"`
+|Object
+|Additional non-action or user id specific event data: e.g: {replication_type: pull_only}
+
+|===
+
+
+== Audit Log Configuration
+
+IMPORTANT: Audit Logging on Sync Gateway is opt-in functionality and is disabled by default.
+
+Admin users can manage Audit logging via the xref:rest-api-admin.adocp[Admin REST API]. 
+Below is a table of possible configuration options.
+
+
+[cols="3*"]
+|===
+
+| Name | Values | Additional Description
+
+| `Destination`
+a|
+
+* Console - The default.
+Outputs the audit events to stdout in JSON format.
+
+* File - Outputs the audit events to the file specified file path in JSON format.
+Audit logs are written to a different location by default than console logs.
+
+| Both active and rotated log files are stored.
+
+| `Max file size`
+| Default - 1GB
+| The maximum size the active audit logging file is allowed to grow before being rotated.
+
+| `Rotation Interval`
+| The time interval in seconds from 900s (15 minutes) to 604800s (7 days) inclusive. 
+| The default is 86400s.
+
+| `Prune age`
+| The time after which Sync Gateway deletes audit rotates audit log files in minutes. 
+| The default is 30 days (43200)
+
+| `Max size`
+| A specified filesize outside of the `Max File Size` property.
+| This is disabled by default.
+
+|===
+
+There are two main ways in which you can configure audit logs:
+
+* Via the bootstrap config.
+This will affect the global configuration.
+
+* Via the database config.
+This will affect the configuration on a per database basis.
+
+For more information on the full configuration schema see:
+
+* xref:configuration-schema-database.adoc[Database Configuration Schema].
+
+* xref:configuration-schema-bootstrap.adoc[Bootstrap Configuration Schema].
+
+=== Configuration with the Bootstrap Config
+
+Below you can find the minimal configuration required in your bootstrap config file to enable audit logging.
+
+[source, json, indent=0]
+----
+
+"logging": {
+    "audit": {
+	  “audit_log_file_path”: “/path/to/file.log”,
+    “enabled”: true,
+    }
+}
+
+----
+
+For further bootstrap configuration options, see xref:configuration-schema-bootstrap.adoc[].
+
+NOTE: Configuration via the bootstrap config will affect your configuration globally.
+
+To successfully configure audit logging, you must set a file path for your audit logs and set the `enabled` parameter to `true`.
+Enabling and disabling audit logging is handled by the Admin REST API at a cluster and Sync Gateway database level.
+Setting enabled to true without any specified enabled events will only enable events with `DefaultEnabled` set to `true`. 
+To view the list of audit events, see xref:audit-log-events.adoc[]
```

Comment:
It doesn't mention here (or elsewhere?) that the audit log will be written to a file named sg_audit.log.